ui/pages/permissions-connect/permissions-connect.container.js

```diff
@@ -54,6 +57,10 @@ const mapStateToProps = (state, ownProps) => {
     subjectType: SUBJECT_TYPES.UNKNOWN,
   };
 
+  ///: BEGIN:ONLY_INCLUDE_IN(flask)
+  const isSnap = targetSubjectMetadata?.subjectType === SUBJECT_TYPES.SNAP;
```

Comment:
Nit: `targetSubjectMetadata` should always be an object (see the `??` operator on the above line), so this `?` isn't needed

```suggestion
  const isSnap = targetSubjectMetadata.subjectType === SUBJECT_TYPES.SNAP;
```